code/game/machinery/telecomms/ntsl2.dm

```diff
@@ -144,17 +172,54 @@ GLOBAL_DATUM_INIT(nttc_config, /datum/nttc_configuration, new())
 
 	// These two stack properly.
 	// Simple job indicator switch.
-	if(toggle_jobs)
-		var/new_name = signal.data["name"] + " ([signal.data["job"]]) "
-		signal.data["name"] = new_name
-		signal.data["realname"] = new_name // this is required because the broadcaster uses this directly if the speaker doesn't have a voice changer on
+	if(job_indicator_type)
+		var/new_name = signal.data["name"]
+		var/job = signal.data["job"]
+		var/job_color = "#000000"
+		if(job in jobs_ai)
+			job_color = "#FF00FF"
+		if(job in jobs_command)
```

Comment:
I'd put else statements after each if.
Looks horrid but will make the code that much faster.
And will work better with the premade job lists

Reply:
Is it possible to have a `switch` for this?
```
switch(job)
    if(jobs_ai)
```
Or something

Reply:
Don't think there is one. Unless you make the lists in the if(...) part. So if("HoP","captain".. etc) but that's gross as well.